doc/src/fix_eco_force.rst

```diff
@@ -0,0 +1,167 @@
+..  index:: fix eco/force
```

Comment:
this is missing full integration into the lists of commands in the manual. please see the output of `make style_check` in the doc folder.